group_vars/all

```diff
@@ -21,7 +21,7 @@ use_pbkdf2: true
 proftpd_sql_auth_type: PBKDF2
 proftpd_files_dir: "{{ galaxy_data }}/database/ftp"
 default_admin_api_key: admin
-postgresql_version: 9.3
+postgresql_version: "{{ '9.3' if ansible_distribution_version <= '15.04' else '9.5' }}"
```

Comment:
I think this comparing strings, not version numbers, so it works until ubuntu 100.X is released, which may give us some time ;). You'd probably still want to use https://docs.ansible.com/ansible/playbooks_tests.html#version-comparison and do `"{{ '9.3' if ansible_distribution_version | version_compare('15.04', '<=') else '9.5' }}"`